src/python/pose_format/utils/holistic.py

```diff
@@ -121,3 +122,56 @@ def load_holistic(frames: list, fps: float = 24, width=1000, height=1000, depth=
                                            additional_holistic_config)
 
     return Pose(header, body)
+
+
+def formatted_holistic_pose(width: int, height: int):
+    dimensions = PoseHeaderDimensions(width=width, height=height, depth=1000)
+    header = PoseHeader(version=0.1, dimensions=dimensions, components=holistic_components("XYZC", 10))
```

Comment:
check if 10 exists in the previous repo

Reply:
No `additional_face_points`, should be removed

Reply:
Oh no, there are 10 if the reduced face points are 128, fixed